.vscode/extensions.json

```diff
@@ -0,0 +1,3 @@
+{
+  "recommendations": ["dbaeumer.vscode-eslint", "esbenp.prettier-vscode", "knisterpeter.vscode-commitizen"]
```

Comment:
This enables auto fix/autocomplete on VSCode for ESLint and Prettier. It's more of a developer experience (DX) improvement.

Reply:
I see! I think we should probably let everyone bring their own editor setup and recommend these plugins in the README under a PR guidelines section or something.
Where possible/required the "squash and merge" will allow us to clean up any commits that don't follow the correct style

Reply:
I agree. I'm going to update the readme with these recommendations 👌.

Now that you mentioned the correct style of commits, we already have it configured on the git hook `commit-msg`. Although this can be bypassed with the `--no-verify` CLI argument.

I'll add a GitHub action to lint commits on pull requests 😉.

Reply:
Just pushed a commit which adds a GitHub action to lint commit messages with the conventional commits specification 👌. 
You can see the two checks bellow.